frontend/src/components/OrganizationDetails.vue

```diff
@@ -0,0 +1,110 @@
+<!--suppress XmlInvalidId -->
+<template>
+  <div id="organization">
+    <v-card outlined class="subgroup">
+      <h2>Organization Details</h2>
+      <v-row no-gutters>
+        <v-col class="col-7">
+          <v-form ref="form">
+            <label for="id" :disabled=" !!organizationId || !editUserDetailsPermission" class="required">Organization ID</label>
```

Comment:
If you remove the ability to update organizations, it should be possible to simplify the code in this file a bit. For example, the OrganizationDetails screen will only be used to create new organizations, so `organizationId` will always be blank. 

Probably all of the code used to disable fields and labels will be removed, actually, as this create organization component will be accessible only to those with the right role, e.g. we won't show the Create Organization button if they're missing the role. 

(On the backend we'll check the role, so even if a user manages to reach the page somehow, they won't be able to create organizations.

Reply:
removed code related to disabling fields on organization Details